test/mpi/threads/pt2pt/Makefile.am

```diff
@@ -20,7 +20,7 @@ noinst_PROGRAMS = threads threaded_sr alltoall sendselfth greq_wait greq_test \
                   mt_probe_sendrecv_huge mt_iprobe_sendrecv_huge \
                   mt_mprobe_sendrecv_huge mt_improbe_sendrecv_huge \
                   mt_sendrecv_pers_huge mt_bsendrecv_pers_huge mt_ssendrecv_pers_huge \
-                  mt_mpicancel
+                  mt_mpicancel mt_perf_pt2pt
 
```

Comment:
Sorry, I think I have a better idea (on the naming and placement) -- How about create a new test folder, `thread/perf` and setup add move this test to `thread/perf/mt_pt2pt_msgrate` ? That way, if we want to introduce more thread performance tests, we have a good place to go and a good naming scheme.

Reply:
Created a new folder for this. Let me know if it looks correct.